telco-examples/mgmt-cluster/airgap/eib/mgmt-cluster-airgap.yaml

```diff
@@ -27,136 +27,88 @@ kubernetes:
-    - name: registry.rancher.com/rancher/hardened-k8s-metrics-server:v0.7.1-build20240401
-    - name: registry.rancher.com/rancher/hardened-kubernetes:v1.30.3-rke2r1-build20240717
-    - name: registry.rancher.com/rancher/hardened-multus-cni:v4.0.2-build20240612
-    - name: registry.rancher.com/rancher/hardened-node-feature-discovery:v0.15.4-build20240513
-    - name: registry.rancher.com/rancher/hardened-whereabouts:v0.7.0-build20240429
-    - name: registry.rancher.com/rancher/helm-project-operator:v0.2.1
-    - name: registry.rancher.com/rancher/k3s-upgrade:v1.30.3-k3s1
-    - name: registry.rancher.com/rancher/klipper-helm:v0.8.4-build20240523
-    - name: registry.rancher.com/rancher/klipper-lb:v0.4.7
-    - name: registry.rancher.com/rancher/kube-api-auth:v0.2.2
-    - name: registry.rancher.com/rancher/kubectl:v1.29.7
-    - name: registry.rancher.com/rancher/local-path-provisioner:v0.0.28
-    - name: registry.rancher.com/rancher/machine:v0.15.0-rancher116
-    - name: registry.rancher.com/rancher/mirrored-cluster-api-controller:v1.7.3
-    - name: registry.rancher.com/rancher/nginx-ingress-controller:v1.10.1-hardened1
-    - name: registry.rancher.com/rancher/prometheus-federator:v0.3.4
-    - name: registry.rancher.com/rancher/pushprox-client:v0.1.3-rancher2-client
-    - name: registry.rancher.com/rancher/pushprox-proxy:v0.1.3-rancher2-proxy
-    - name: registry.rancher.com/rancher/rancher-agent:v2.9.1
-    - name: registry.rancher.com/rancher/rancher-csp-adapter:v4.0.0
-    - name: registry.rancher.com/rancher/rancher-webhook:v0.5.1
-    - name: registry.rancher.com/rancher/rancher:v2.9.1
-    - name: registry.rancher.com/rancher/rke-tools:v0.1.100
-    - name: registry.rancher.com/rancher/rke2-cloud-provider:v1.29.3-build20240515
-    - name: registry.rancher.com/rancher/rke2-runtime:v1.30.3-rke2r1
-    - name: registry.rancher.com/rancher/rke2-upgrade:v1.30.3-rke2r1
-    - name: registry.rancher.com/rancher/security-scan:v0.2.16
-    - name: registry.rancher.com/rancher/shell:v0.2.1
-    - name: registry.rancher.com/rancher/system-agent-installer-k3s:v1.30.3-k3s1
-    - name: registry.rancher.com/rancher/system-agent-installer-rke2:v1.30.3-rke2r1
-    - name: registry.rancher.com/rancher/system-agent:v0.3.8-suc
-    - name: registry.rancher.com/rancher/system-upgrade-controller:v0.13.4
-    - name: registry.rancher.com/rancher/ui-plugin-catalog:2.0.1
-    - name: registry.rancher.com/rancher/kubectl:v1.20.2
+    - name: registry.suse.com/rancher/cluster-api-controller:v1.8.4
+    - name: registry.suse.com/rancher/cluster-api-provider-rke2-bootstrap:v0.9.0
+    - name: registry.suse.com/rancher/cluster-api-provider-rke2-controlplane:v0.9.0
+    - name: registry.rancher.com/rancher/cluster-api-metal3-controller:v1.8.2
+    - name: registry.rancher.com/rancher/cluster-api-metal3-ipam-controller:v1.8.1
+    - name: registry.rancher.com/rancher/cluster-api-operator:v0.14.0
```

Comment:
Is this everything necessary for Metal3 + CAPI? What about baremetal-operator for example?

Reply:
Same... I will re-test it

Reply:
The same, the chart metal3 is downloading all images into hauler:
```
-rw-r--r--. 1 root root  52893120 Jan 21 10:21 registry.suse.com_edge_3.2_baremetal-operator:0.8.0-registry.tar.zst
-rw-r--r--. 1 root root 684974888 Jan 21 10:21 registry.suse.com_edge_3.2_ironic-ipa-downloader:3.0.0-registry.tar.zst
-rw-r--r--. 1 root root 221733244 Jan 21 10:21 registry.suse.com_edge_3.2_ironic:26.1.2.0-registry.tar.zst
```
so no need more images to be added to embeddedartifacts